src/Symfony/Bridge/PhpUnit/Tests/DeprecationErrorHandler/DeprecationTest.php

```diff
@@ -11,11 +11,35 @@
 
 namespace Symfony\Bridge\PhpUnit\Tests\DeprecationErrorHandler;
 
+use Composer\Autoload\ClassLoader;
 use PHPUnit\Framework\TestCase;
 use Symfony\Bridge\PhpUnit\DeprecationErrorHandler\Deprecation;
+use Symfony\Bridge\PhpUnit\Legacy\SymfonyTestsListenerForV5;
 
 class DeprecationTest extends TestCase
 {
+    public static function setUpBeforeClass(): void
+    {
+        // Use Deprecation class of the repository. Without that, Deprecation
+        // class from vendors that is not up to date during PR is used
+        require_once __DIR__.'/../../DeprecationErrorHandler/Deprecation.php';
```

Comment:
For what method of running the test suite is this required? I probably isn't for running it component by component, right?

Reply:
I don't know which method is involved but this is needed for the CI method (at least for AppVeyor).

Reply:
What happens when I work on this component is that we sometimes merge even if the CI is red :sweat_smile: I think @nicolas-grekas runs the tests locally before though

Reply:
Probably :) I personally use a local script which copy repository files to vendor directories...

Reply:
My point is, this should probably be removed right before merging.

Reply:
Maybe you're right... I'm preparing a PR especially for this fix anyway.